src/components/StaticThought.tsx

```diff
@@ -82,7 +82,7 @@ const StaticThought = ({
         />
       )}
 
-      <Superscript simplePath={simplePath} showContexts={showContexts} superscript={false} />
```

Comment:
Not sure what was the reason behind passing `false` for `superscript`. I tried testing as much as I could figure out. Please let me know if there was a specific reason in setting this to false.

Reply:
I did git blame all the way back a few years and still couldn't find the reason for it. I can see that whoever wrote it (which could have been me) wanted to still render the superscript container, even if the superscript was not supposed to be rendered. But I am not sure the reason. 

We may be able to remove it completely, but I would have to do some more rigorous testing to find out for sure.

I would probably leave it as-is until we fix the current issue, just to make sure we are not creating a regression.